File: Cracking Coding Interview/Through C#/LearningToSolving/src/Module-1/Assignment/Task_1.cs
namespace LearningToSolving.Src.Module_1.Assignment;

public class Task_1 {
    
    // Calculate the factorial
    public void Solution()
    {
        Console.Write("Enter input number: ");
        uint input_number = Convert.ToUInt32(Console.ReadLine());

        uint result = 1;

        while (input_number > 1)
        {
            result *= input_number;
            input_number--;
        }

        Console.WriteLine("The factorial result is " + result);
    }
}

File: Cracking Coding Interview/Through PHP/Module-1/Assignment/Task_1.php
<?php
namespace Module_1;

class Task_1 {
    public function Solution() {
        $input_number = readline("Enter input number : ");
        $result = 1;

        while ($input_number > 1) {
            $result *= $input_number;
            $input_number--;
        }

        printf("The result of factorial is " . (int)$result);
    }
}

$tsk_1 = new Task_1();
$tsk_1->Solution();